manifests/pipelines/aiedge-e2e.pipeline.yaml

```diff
@@ -136,7 +136,22 @@ spec:
     - input: "$(params.fetch-model)"
       operator: in
       values: ["git"]
-  # Use kserve-download-model container image to fetch from S3 storage
+  - name: cleanup-git-fetch
```

Comment:
Maybe `move-model-to-root-dir` would be better? Or something like that.

Reply:
Sure NP, I had no idea what to call it